chart/values.yaml

```diff
@@ -0,0 +1,237 @@
+# Default values for kube-httpcache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: quay.io/mittwald/kube-httpcache
+  pullPolicy: IfNotPresent
+  tag: "stable"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Enable StatefulSet or Deployment
+statefulset:
+  enabled: true
+
+deployment:
+  enabled: false
+
+# Enable configMap for Varnish Template File (see below defaultConfig)
+# OR use extravolume with name "template" if the file is too big
+configmap:
+  enabled: true
+
+# kube-httpcache specific configuration
+cache:
+  # name of frontend service
+  frontendservice: kube-httpcache
+  # name of backend service
+  backendservice: backend-service
```

Comment:
Currently, it is assumed that the `backend-service` lives in the same namespace as the operator.

I guess we could optionally support a `backendServiceNamespace` which defaults to the currently used `$(NAMESPACE)` environment variable if it is not set.

Reply:
fixed